rectools/models/bert4rec.py

```diff
@@ -77,16 +79,26 @@ def _collate_fn_train(
             y[i, -len(ses) :] = target  # ses: [session_len] -> y[i]: [session_max_len + 1]
             yw[i, -len(ses) :] = ses_weights  # ses_weights: [session_len] -> yw[i]: [session_max_len + 1]
 
-        return torch.LongTensor(x), torch.LongTensor(y), torch.FloatTensor(yw)
-
-    def _collate_fn_recommend(self, batch: List[Tuple[List[int], List[float]]]) -> torch.LongTensor:
+        # TODO: we are sampling negatives for paddings. Let's think about it later
+        batch_dict = dict({"x": torch.LongTensor(x), "y": torch.LongTensor(y), "yw": torch.FloatTensor(yw)})
```

Comment:
How is `dict({"key": "value"})` better then `{"key": "value"}`? :)
Here and everywhere

Reply:
fixed